project-base/storefront/components/Layout/Header/MenuIconic/SalesRepresentative.tsx

```diff
@@ -0,0 +1,69 @@
+import { MailIcon } from 'components/Basic/Icon/MailIcon';
+import { SalesPhoneIcon } from 'components/Basic/Icon/SalesPhoneIcon';
+import { Image } from 'components/Basic/Image/Image';
+import { useCurrentCustomerData } from 'connectors/customer/CurrentCustomer';
+import { TypeImage } from 'graphql/types';
+import useTranslation from 'next-translate/useTranslation';
+
+export const SalesRepresentative: FC = () => {
+    const { t } = useTranslation();
+    const currentCustomerData = useCurrentCustomerData();
+    const salesRepresentative = currentCustomerData?.salesRepresentative;
+
+    if (!salesRepresentative) {
+        return null;
+    }
+
+    const { telephone, email } = salesRepresentative;
+
+    return (
+        <div className="flex items-start gap-4 w-full pt-4">
+            {salesRepresentative.images.length && (
+                <Image
+                    alt={t('Need advice?')}
+                    className="w-12 h-12 rounded-full"
+                    height={100}
+                    src={getRandomImage(salesRepresentative.images).url}
+                    width={100}
+                />
+            )}
+            <div className="font-medium dark:text-white">
+                <p className="text-lg font-semibold font-secondary leading-7">
+                    {getFullname(salesRepresentative.firstName, salesRepresentative.lastName)}
+                </p>
```

Comment:
If `getFullName` returns an empty string, this paragraph is still going to be present.

Reply:
Changed it to `null | undefined` instead and added check for nullish value.